code/game/objects/effects/mines.dm

```diff
@@ -133,8 +133,10 @@
 	victim.drop_r_hand()
 	victim.drop_l_hand()
 	victim.put_in_hands(chainsaw)
-
+	chainsaw.attack_self(victim)
+	chainsaw.wield(victim)
 	victim.reagents.add_reagent("adminordrazine",25)
+	victim.reagents.add_reagent("synaptizine",25)

```

Comment:
This is literally pointless when adminordrazine already sets your weakened, paralysis, and stunned to 0---the only thing this is accomplishing is giving a 30% chance to do 1 tox damage, which will be healed the very next time adminordrazine processes